web/src/components/package/PackageDetail.vue

```diff
@@ -65,6 +65,7 @@
                   <font-awesome-icon class="icon-margin-right" :icon="['fas', 'box-open']" />Package info
                 </p>
                 <div class="panel-block">Author: {{ packageInfo.owner }}</div>
+                <div v-if="packageInfo.repository !== ''" class="panel-block"><a href="{{ packageInfo.repository }}">Repository</a></div>
```

Comment:
```suggestion
                <div v-if="packageInfo.repository !== ''" class="panel-block"><a href="{{ packageInfo.repository }}" class="has-text-primary">Repository</a></div>
```

Perhaps add a class `is-primary` to the inner anchor tag so that the color of this matches with the color scheme of the page.